scripts/zones/La_Theine_Plateau/npcs/Dimensional_Portal.lua

```diff
@@ -2,19 +2,18 @@
 -- Area: La Theine Plateau
 --  NPC: Dimensional Portal
 -----------------------------------
+local ID = require("scripts/zones/La_Theine_Plateau/IDs")
 require("scripts/globals/keyitems")
```

Comment:
Looks like this was a c/p of the Konschtat one with a KI require instead of missions.

(Tahrongi is fine.)